cmake/features.cmake

```diff
@@ -44,6 +44,8 @@ endif()
 
 ov_dependent_option (ENABLE_ONEDNN_FOR_GPU "Enable oneDNN with GPU support" ${ENABLE_ONEDNN_FOR_GPU_DEFAULT} "ENABLE_INTEL_GPU" OFF)
 
+ov_option (ENABLE_INTEL_NPU "NPU plugin for OpenVINO runtime" ON)
```

Comment:
We are preparing now the changes we need in VPUX CI pipeline in order to keep this OFF by default.

Reply:
Set to OFF now